### PR TITLE
Docker image - update

### PR DESCRIPTION
**Pull request by stenczelt**
Updated build for the minimal docker image

### CODE REVIEW

**Comment by stenczelt** [comment]
@jameskermode this is failing in the PyPI build stage already. Have you seen these issues? If I decoded the output correctly then the multibuild's setup scripts are doing something fishy perhaps?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -6,24 +6,24 @@ on:
...
     runs-on: ${{ matrix.os }}
-    
+
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest]
-        python: [3.6, 3.7, 3.8, 3.9]
-        manylinux: [2010, 2014]
-        plat: [x86_64]
+        os: [ ubuntu-latest, macos-latest ]
```
Comment (jameskermode) [inline .github/workflows/build.yml:23]: you could try reverting these whitespace changes in case they are what is breaking the wheel builds

**Comment by jameskermode** [comment]
Comparing with the most recent successful build on `main`, logs for which can be found at https://github.com/libAtoms/quippy-wheels/runs/3221779714?check_suite_focus=true#step:4:62, suggests that perhaps `ubuntu-latest` has changed and this is what's breaking the build. I've triggered a rebuild on `main` at https://github.com/libAtoms/quippy-wheels/actions/runs/1171675012 to check this hypothesis.

**Comment by jameskermode** [comment]
Nope, still works fine on `main` so it must be something you've changed.

**Comment by jameskermode** [comment]
Closing in preparation for archiving repo.

**Pull request closed without merging**